sdk/websites/Microsoft.Azure.Management.WebSites/src/Generated/Models/SkuCapacity.cs

```diff
@@ -33,14 +33,17 @@ public SkuCapacity()
         /// Service plan SKU.</param>
         /// <param name="maximum">Maximum number of workers for this App
         /// Service plan SKU.</param>
+        /// <param name="elasticMaximum">Maximum number of Elastic workers for
+        /// this App Service plan SKU.</param>
         /// <param name="defaultProperty">Default number of workers for this
         /// App Service plan SKU.</param>
         /// <param name="scaleType">Available scale configurations for an App
         /// Service plan.</param>
-        public SkuCapacity(int? minimum = default(int?), int? maximum = default(int?), int? defaultProperty = default(int?), string scaleType = default(string))
+        public SkuCapacity(int? minimum = default(int?), int? maximum = default(int?), int? elasticMaximum = default(int?), int? defaultProperty = default(int?), string scaleType = default(string))
```

Comment:
`int? elasticMaximum = default(int?)` should be after `scaleType` since it is a new parameter.